extensions/eclipse-che-theia-user-preferences/src/node/che-theia-preferences-synchronizer.ts

```diff
@@ -0,0 +1,86 @@
+/********************************************************************************
+ * Copyright (C) 2018-2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/// <reference types="@theia/core/src/typings/nsfw/index"/>
+
+import { injectable, inject } from "inversify";
```

Comment:
somehow we have a missing formatter/linter as some imports are with double quotes and some other with single quotes

Reply:
Indeed. Will check.

Reply:
Done, thanks